src/containers/Storage/StorageGroups/StorageGroups.tsx

```diff
@@ -120,6 +120,24 @@ function StorageGroups({data, tableSettings, visibleEntities, nodes}: StorageGro
             align: DataTable.LEFT,
             defaultOrder: DataTable.DESCENDING,
         },
+        {
+            name: TableColumnsIds.UsedPercents,
+            header: tableColumnsNames[TableColumnsIds.UsedPercents],
+            width: 100,
+            render: ({row}) => {
+                const usage = getUsage(row, 5);
+                return (
+                    <Label
+                        theme={getUsageSeverity(usage)}
+                        className={b('usage-label', {overload: usage >= 100})}
+                    >
+                        ≥ {usage}%
```

Comment:
It seems it looks strange. Just as example: my calculated usage equals to 20%. And I see on screen >=20%. So... is it 21 or 99 percent?
Maybe `~` symbol fits better?

Reply:
Good catch. I thought about it. It will be clear in comparison to other rows. But in case with few rows this is still an issue.

This is how it is shown in design though. I would keep it this way for now and discussed with the team once again.